test/test.js

```diff
@@ -1545,6 +1545,22 @@ exports['wrap EventEmitter (or jQuery) on handler with args wrapping by array']
     });
 };
 
+exports['takeUntil'] = function (test) {
+    a = _();
+    b = _();
+    a.resume();
```

Comment:
Nope. The callback will only be called once. `toArray` will wait until the source stream (`c` in this case) ends before emitting everything it saw.